packages/headless/src/utils/validate-payload.ts

```diff
@@ -41,6 +41,18 @@ export const optionalNonEmptyVersionString = new StringValue({
   regex: /^\d+\.\d+\.\d+$/,
 });
 
+export const optionalTrackingId = new StringValue({
+  required: false,
+  emptyAllowed: false,
+  regex: /^[a-zA-Z0-9_\-.]{1,100}$/,
+});
+
+export const requiredTrackingId = new StringValue({
+  required: false,
```

Comment:
- `required: false`?
- Check the regex here, I think you are missing a `\` before the `.`. Same for the one above

Reply:
No need for escape in []:
https://stackoverflow.com/questions/19976018/does-a-dot-have-to-be-escaped-in-a-character-class-square-brackets-of-a-regula

Reply:
I was mindblown too lol 
![image](https://github.com/user-attachments/assets/b662df35-5323-44ed-ab03-400e9a7e562a)

Reply:
@fvanham-coveo wow... for the required I SWEAR I specifically put true there xD.

Thanks for catching that!